src/components/advert-test.js

```diff
@@ -40,49 +51,49 @@ export default function (rootElement) {
       config.props.push('whatever-from-covatic');
       resolve();
     });
-
   }
 
-  function initSlots() {
-    return new Promise((resolve, reject) => {
-      // get slots from page
-      config.slots = Array.from(document.querySelectorAll('.advert-slot')).map((slot) => {
-        const slotConfig = Object.assign({}, slotDefaults);
+  loadGPT()
+    .then(() => getCovatic())
+    .then(() => getPeer39())
```

Comment:
```suggestion
    .then(Promise.all(getCovatic(), getPeer39()))
```
I wonder if we could do this to have these load together?